language/de.json

```
@@ -23,7 +23,7 @@
     },
     {
       "label": "Steuerelemente aktivieren",
-      "description": "Dies gibt dem Nutzer etwa die Möglichkeit, das Abspielen zu pausieren."
+      "description": "Dies gibt dem Nutzer z.B. die Möglichkeit, das Abspielen zu pausieren."
```

```suggestion
      "description": "Dies gibt dem Nutzer z. B. die Möglichkeit, das Abspielen zu pausieren."
```
It's one of the most common mistakes in written German to omit the space character between _z._ and _B._. The correct spelling is _z. B._ [according to the Duden](https://www.duden.de/rechtschreibung/z__B_) -- and also that's quite logical for an abbreviation IMHO.

---

I've corrected the mistake.